wherescape/connectors/hubspot/process_data.py

```diff
@@ -0,0 +1,155 @@
+import logging
+from decimal import Decimal
+from datetime import datetime
+from .hubspot_wrapper import Hubspot
+
+# from helper_functions import compare_names
+
+"""
+This module processes the collected data so it can be send to the Hubspot Module
+"""
+
+
+def hubspot_process_results(
+    access_token: str, results: list, column_names: list, table_name: str
+):
+    """
+    function that handles the processing of the results for it to be send to Hubspot
```

Comment:
Comments are starting with both capitals & no-caps. Prefer to use capitals everywhere. 
(NB. if the level of comments is this, the rest must be good!)